frame/system/src/lib.rs

```diff
@@ -370,23 +369,6 @@ pub mod pallet {
 
 	#[pallet::call]
 	impl<T: Config> Pallet<T> {
-		/// A dispatch that will fill the block weight up to the given ratio.
-		// TODO: This should only be available for testing, rather than in general usage, but
-		// that's not possible at present (since it's within the pallet macro).
-		#[pallet::weight(*_ratio * T::BlockWeights::get().max_block)]
-		pub fn fill_block(origin: OriginFor<T>, _ratio: Perbill) -> DispatchResultWithPostInfo {
```

Comment:
this breaks transaction version.
can we have a dummy call to preserve the call index of the other calls to reduce the impact?

Reply:
I recall there were some discussion about specify call index to avoid this kind of issue but can't find it anymore. That will be the proper solution.

For now, maybe just `pub fn removed_call_do_not_use(origin: OriginFor<T>, _ratio: Perbill)`

Reply:
@xlc do we even need `_ratio` as an argument?

Reply:
we don't need the argument.

@xlc is this a big deal though? most apps should handle this dynamically. For others, in any case, almost all releases contain a tx-version breaking change (bth we are not even bumping it strictly).

I would say let's break it and move on.

Reply:
@kianenigma I removed the dummy call then.